app/views/calls/_new_call.html.erb

```diff
@@ -1,25 +1,14 @@
-<div class="modal fade new-call" tabindex="-1" role="dialog" id="call-<%= patient.primary_phone_display %>">
-  <div class="modal-dialog">
-    <div class="modal-content">
-      <div class="modal-body">
-        <div class="calls-layout">
-          <button type="button" class="close close-terms" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-          <h4 class="modal-title calls-request">Call <b><%= patient.name %></b> now:</h4>
-          <br>
-            <!-- wrapper for ajaxing when recording new call  -->
-            <%= display_other_contact_and_phone_if_exists(patient) %>
 
-            <%= content_tag :h4, patient.primary_phone_display, class: 'calls-phone' %>
+Call <b><%= patient.name %></b> now:</h4>
```

Comment:
Looks like the opening of the h4 tag here is in `_modal`. Can we move that in here?